rfc047-release-process.md

```diff
@@ -44,40 +44,51 @@ Additional labels for pre-release and build metadata are available as extensions
 Chef Client specific clarifications:
 
 ```
-Given a version number MAJOR.MINOR.PATCH, increment the:
+Given a version number MAJOR.MINOR.BUILD:
 
-MAJOR version when the release will break existing functionality
-MINOR version when you add any new features
-PATCH version when you are only fixing bugs, typically regressions
+Increment the MAJOR version when the release will break existing functionality
+Increment the MINOR version when you add any new features
+The BUILD version is increased automatically by the build system
```

Comment:
@jkeiser and I talked about that and figured we'd leave it up to the implementation. That's the obvious choice, but maybe it's easier to keep incrementing like Microsoft does for a while like:

```
12.0.123
12.0.124
12.1.125
12.1.126
12.2.127
13.0.128
```

I would guess it'd be not much difference in effort to implement it either way. I didn't have a compelling reason to specify that behavior besides the convenience of knowing.